flatbuffer-schemes/README.md

```diff
@@ -0,0 +1,18 @@
+## Description
+
+Directory for files related to socket communication
+
+## How To
+### Generate Java or Rust based on fbs schema?
+
+#### Requirements
+[flatcc](https://flatbuffers.dev/flatbuffers_guide_building.html) version 23.5.26.
+
+#### Instructions
+
+Java
+`$ flatcc --java <file>.fbs`
```

Comment:
The command should start with "flatc" and not with "flatcc"
![Screenshot from 2024-06-27 11-25-09](https://github.com/Pending-Name-21/miscellany/assets/135136456/5333750d-ee50-4ce9-9d64-d5b98acbc95a)